coala_quickstart/Constants.py

```diff
@@ -1,14 +1,14 @@
 IMPORTANT_BEAR_LIST = {
     'All': {'FilenameBear', 'InvalidLinkBear', 'LineCountBear'},
-    "C": {"GNUIndentBear", 'CSecurityBear', 'ClangComplexityBear'},
+    'C': {'GNUIndentBear', 'CSecurityBear', 'ClangComplexityBear'},
     'C#': {'CPDBear', 'CSharpLintBear', 'SpaceConsistencyBear'},
     'C++': {'GNUIndentBear', 'CPDBear', 'CPPCheckBear', 'CPPCleanBear',
             'ClangComplexityBear'},
     'CMake': {'CMakeLintBear', 'SpaceConsistencyBear'},
     'CSS': {'CSSLintBear', 'SpaceConsistencyBear'},
-    "JavaScript": {"JSHintBear", "JSComplexityBear"},
-    "Java": {"JavaPMD", "CheckstyleBear"},
-    "Python": {"PycodestyleBear"}}
+    'JavaScript': {'JSHintBear', 'JSComplexityBear'},
+    'Java': {'JavaPMD', 'CheckstyleBear'},
+    'Python': {'PycodestyleBear'}}
```

Comment:
if you are around , an optional fix is moving the final `}` to a new line and adding a trailing comma.
but optional..